src/components/Game/Game.tsx

```diff
@@ -51,11 +51,32 @@ export const Game: FunctionalComponent<GameProps> = ({
     onSubmit(word);
     setWord("");
   }, [word, setWord, onSubmit]);
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      const key = event.key.toLowerCase();
+      if (key === "backspace") {
+        handleDelete();
+      }
+      else if (key === "enter" && word.length > 0) {
+        handleSubmit();
+      }
```

Comment:
Could you add a case here to call handleClear on escape?

Reply:
Added.